share/spice/hex/hex.js

```diff
@@ -84,4 +80,8 @@
             });
         });
     };
-}(this));
\ No newline at end of file
+
+    Handlebars.registerHelper('hex_plural', function(number, options) {
```

Comment:
@milmazz whoops, you're right that doesn't help in this case. 

I thought we had another function that does what you're expecting and I just found it! 

https://github.com/duckduckgo/duckduckgo-utils/blob/master/util.js#L477

`DDG.pluralize()` simply returns the word, but it's not a handlebars helper. You could use it in the `normalize` if you like?